_posts/2016-09-12-schedule.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Schedule Released!"
+authors:
+ - badboy
+ - hoverbear
+---
+
+At long last, a schedule!
+
+We're proud to announce our lineup for RustFest 2016 in Berlin!
+
+[**Click here to view the schedule!**](/schedule)
+
+# Saturday
+
+Starting at 9:00 and going until 19:00, Saturday is full of talks at the Microsoft Atrium!
+
+On Saturday your ticket includes lunch, dinner, coffee and cold drinks throughout the day.
+
+# Sunday
+
+Starting at 10:00 and going until 17:30, Sunday features keynotes to start and end the day as well as workshops start!
+
+Workshops are located at Co.up and ThoughtWorks Werkstatt which are a short transit trip. You can check out the [transport](/blog/Transportation-at-the-Conference) post to see how to get to them!
+
+Workshop venues are equipped with light refreshments.
```

Comment:
Co-up has drinks in fridges but they aren't free usually. We need to leave a sheet there asking people to sign in when they pick something and we give Co-Up a lump sum about all drinks after. I think we tell them, we'll do that, that should be fine.